### DISCUSSION TITLE
Add Support for Vencord

### DISCUSSION THREAD

**Discussion by RarelySam** [victornpb/undiscord#574] [3. Ideas]
Add Support for Vencord

**Comment by dxdvostfr** [comment]
yes, please submit your script to vencord so they can add it as plugin, it would be very useful to use undiscord directly in-app

**Comment by Razor7100** [comment]
If not you can add custom user plugins to vencord now so they just gotta make it a plugin for someone to install.

**Comment by oldmat3** [comment]
this would be great
